Sources/MongoSwift/MongoClient.swift

```diff
@@ -278,6 +278,96 @@ public class MongoClient {
         return MongoDatabase(name: name, client: self, options: options)
     }
 
+    /**
+     * Starts a `ChangeStream` on a `MongoClient`. Allows the client to observe all changes in a cluster - excluding
+     * system collections i.g. "config", "local", and "admin" databases. By default, the type `CollectionType` is
```

Comment:
Since we're on a client here, there is no such `CollectionType`, so that sentence can be removed.